src/FluffySpoon.AspNet.LetsEncrypt.Azure/AzureAppServiceSslBindingCertificatePersistenceStrategy.cs

```diff
@@ -249,5 +323,37 @@ private async Task<IAppServiceCertificate> GetExistingCertificateAsync(Persisten
 
 			return null;
 		}
+
+		private async Task<X509Certificate2> GetExistingCertificateAsync(PersistenceType persistenceType)
+		{
+			var azureCert = await GetExistingAzureCertificateAsync(persistenceType);
+
+			X509Store certStore = new X509Store(StoreName.My, StoreLocation.CurrentUser);
```

Comment:
**Optional:** Use `var` here to be consistent. The same applies for the other places where this is the case.

Reply:
Changed.